haskell-language-server.cabal

```diff
@@ -151,7 +153,7 @@ executable haskell-language-server
      -- which works for now.
     , ghc
      --------------------------------------------------------------
-    , ghc-check
+    , ghc-check ^>= 0.1
```

Comment:
There has just been a new release of this, with bug fixes. Perhaps set the lower bound to that already? I think 0.3?

Reply:
Not in this PR, we only support `0.1.*` at the moment and I am waiting until ghcide has merged it: https://github.com/digital-asset/ghcide/pull/535

Reply:
ok